### PR TITLE
Remove NCEG operators from the language

### PR DESCRIPTION
**Pull request by JinShil**
These were made an error long ago in 2.072 (See https://dlang.org/deprecate.html).  It's always nice to keep the shop clean.

Spec PR: https://github.com/dlang/dlang.org/pull/2280

### CODE REVIEW

**Review by jacob-carlborg — APPROVED** [review]

**Comment by ibuclaw** [comment]
Not so sure about removing support from the backend though, there may be another frontend that uses it.

File: src/dmd/backend/evalu8.c [hunk 1]
```diff
@@ -1599,63 +1599,55 @@ elem * evalu8(elem *e, goal_t goal)
...
         // BUG: complex numbers
         i ^= d1 !<>= d2;
         e->EV.Vint = i;
         break;
 
     case OPnlg:
         i++;
-    case OPlg:
         // BUG: complex numbers
         i ^= d1 <> d2;
```
Comment (ibuclaw) [inline src/dmd/backend/evalu8.c]: Infact this suggests that dmc supports nceg operators still, so definitely don't remove backend support.

**Review by ibuclaw — DISMISSED** [review]

**Comment by JinShil** [comment]
Backend support has been restored.

File: src/dmd/tokens.d [hunk 2]
```diff
@@ -94,18 +94,6 @@ enum TOK : int
...
-    unord,
-    lg,
-    leg,
-    ule,
-    ul,
-    uge,
-    ug,
-    ue,
-
     // 72
```
Comment (wilzbach) [inline src/dmd/tokens.d]: I guess this number should be removed too as it gets outdated too easily.

**Review by wilzbach — APPROVED** [review]
Nice! Thanks a lot for cleaning up the legacy baggage.

File: src/dmd/tokens.d (hunk 2 above)
Comment (JinShil) [reply src/dmd/tokens.d]: I subtracted 8 from each of the numbers to try and keep the status quo.

**Comment by WalterBright** [comment]
> Backend support was restored

Thank you, it's used by DMC/C++.

**Comment by JinShil** [comment]
Closing and reopening in an attempt to wake up Semaphore CI.

**Pull request merged**